examples/iscsi-pd/iscsi-pd.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "iscsipd",
+    "kind": "Pod",
+    "apiVersion": "v1beta1",
```

Comment:
Please include this in examples_test.go.

Also, all other examples are being remade into v1beta3.  Is it not time for this one, too?